docs/docs/docs/01-core/content-releases/01-backend.md

```diff
@@ -56,6 +56,19 @@ packages/core/content-releases/server/src/routes/release.ts
   }
   ```
 
+**Get all releases with/without an entry**:
+
+- method: `GET`
+- endpoint: `/content-releases/`
+- params:
+  ```ts
+  {
+    contentTypeUid: string;
+    entryId: number;
```

Comment:
Will entryId still be correct after your refactor for single-types? Aren't we moving to something like entryUid or entrySlug?